### PR TITLE
Rename database destroying "deploy" command

### LINKED ISSUE #581: Rename hsctl deploy subcommand
"deploy" is a confusing name for a command that deletes your database.

### PR DESCRIPTION
**Pull request by ironfroggy**
Changed hsctl deploy command to a hsctl rebuild --db to make deleting your database harder and less badly named.

Closes #581

### CODE REVIEW

**Comment by mjstealey** [comment]
Might add "install" to have the same function as "rebuild --db" as that would seem reasonable naming wise.

**Comment by mjstealey** [comment]
Also may want to update this: https://github.com/hydroshare/hydroshare/blob/581-rename-deploy/hsctl#L109

To account for this: https://github.com/hydroshare/hydroshare/pull/618/files#diff-1d9fbaa49f3ea3ec9956ffcd21ac7629L410

Based on your updated code that statement should no longer be necessary.

File: hsctl [hunk 1]
```diff
@@ -227,16 +227,26 @@ deploy_hs() {
...
-    exit 1
+	if [ $1 != "rebuild" ] || [ $2 != "--db" ]
+	then
+	    display_usage
+	    exit 1
+	fi
 fi
 
+rebuild_hs() {
+	if [ "$2" = "--db" ]
```
Comment (hyi) [inline hsctl]: Just wondering whether the "==" symbol should be used here rather than "="

**Comment by ironfroggy** [comment]
I have made @hyi's suggested update.
@mjstealey the update you suggest is actually part of #300 and I've fixed it in that PR which seems more appropriate.

**Comment by mjstealey** [comment]
:+1: - Looks fine to me, just need to inform the team once this is pulled in and update the Jenkins script to use **rebuild --db** instead of deploy. I also still like **install** as the same function as it makes sense when starting from thin air.

**Pull request merged**